cabal-install/src/Distribution/Client/ProjectPlanning.hs

```diff
@@ -4300,8 +4299,43 @@ setupHsTestFlags (ElaboratedConfiguredPackage{..}) _ verbosity builddir =
     , testKeepTix = toFlag elabTestKeepTix
     , testWrapper = maybe mempty toFlag elabTestWrapper
     , testFailWhenNoTestSuites = toFlag elabTestFailWhenNoTestSuites
+    , testCoverageLibsModules = toFlag covIncludeModules
+    , testCoverageDistPrefs = toFlag covLibsDistPref
```

Comment:
Need to consider how this interacts if `./Setup` is built with older version of `Cabal` library.